bats/sanity-check/25-cpu-topology.bats

```diff
@@ -525,3 +525,37 @@ selftest_cpuset_negated() {
     test_yaml_regexp "/tests/0/threads/3/messages/0/text" "I> ${cpuset[3]}\$"
     test_yaml_regexp "/tests/0/threads/3/messages/1/text" "I> ${cpuset[3]}\$"
 }
+
+@test "selftest test smt skip" {
+
+    declare -A yamldump
+    # if all thread ids are 0(--cpuset=t0) skip the test
+    sandstone_selftest -e selftest_test_requires_smt --cpuset=t0
+
+    if (( status != 0 )); then
+        skip "Test only works with Debug builds (to mock the topology) or multi-socket systems"
+    fi
```

Comment:
Remove these lines, this should not be a problem and you didn't set the mock topology anyway. The only problem would be on a hypothetical system with only non-zero thread IDs, which I don't think exist.